designs/virtualization/4.x/snapshots/snapshots.md

```diff
@@ -0,0 +1,85 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# VM snapshots
+
+This item will cover updates to vm-snapshots
+- Creating a VM snapshot
+- Snapshots on the VM details page
+- Virtualization snapshot list
+- Restoring a VM from snapshot
+
+## Creating a snapshot
+
+![Virtualization - VMs list](img/0-0.jpg)
+From the virtual machines menu item, the user selects “Create Snapshot” from a VM item’s kebab dropdown.
+The same action can be taken from a VM details Action dropdown.
+
+![Virtualization - turn off VM notification](img/0-1.jpg)
```

Comment:
Can we confirm that the icon is meant to be at the top like this and not beside the content of the body? In the conventions it shows the latter.

Reply:
@itsptk can you confirm that the icon is meant to be on the top next to the title?
I believe I saw this pattern on the operators' domain
Thanks

Reply:
ah yes I see it for the operator here when it's being uninstalled
I do question though if this is the way it will be implemented when the modals from PF4 get pulled in. They don't current have support for adding icons to the heading like this (But we could ask for it). @itsptk have you talked to the PF side about this at all?
<img width="641" alt="Screen Shot 2020-07-08 at 4 43 43 PM" src="https://user-images.githubusercontent.com/287880/86963030-3f26e900-c13a-11ea-88c0-18f088ef01e7.png">

Reply:
follow up - looks like this is on the books for PF. So we should be ok to continue with these designs.
https://github.com/patternfly/patternfly-design/issues/863
Seems like a nice addition to the alert convention :)

Reply:
Sorry for delay but yeah, as that PF issue mentions it was the recommended method in the design guidelines for destructive confirmation modals: https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/modal#confirm-a-destructive-action